docs/docs/authentication/users.mdx

```diff
@@ -139,7 +135,7 @@ await nhost.auth.signUp({
 
 ## Import Users
 
-If you have users in a different system, you can import them into Nhost. When importing users you should insert the users directly into the database instead of using the authentication endpoints (`/signup/email-password`).
+If you have users in a different system, you can import them into Nhost. When importing users you should insert the users directly into the database instead of using the authentication endpoints (`/signup/email-password`) to avoid sending potential transactional emails.
```

Comment:
```suggestion
If you have users in a different system, you can import them into Nhost. When importing users you should insert the users directly into the database instead of using the authentication endpoints (`/signup/email-password`) to avoid sending unnecessary transactional emails.
```